dist/summernote.js

```diff
@@ -5892,12 +5894,12 @@
       var $container = options.dialogsInBody ? $(document.body) : $editor;
 
       var body = '<div class="form-group">' +
-                   '<label>' + lang.link.textToDisplay + '</label>' +
-                   '<input class="note-link-text form-control" type="text" />' +
+                   '<label id="Display">' + lang.link.textToDisplay + '</label>' +
+                   '<input class="note-link-text form-control" type="text" aria-labelledby="Display" />' +
                  '</div>' +
                  '<div class="form-group">' +
-                   '<label>' + lang.link.url + '</label>' +
-                   '<input class="note-link-url form-control" type="text" value="http://" />' +
+                   '<label id="url">' + lang.link.url + '</label>' +
```

Comment:
You may want to avoid using very broad ids, as they may already exist on the page where summernote is being used and ids are supposed to be unique across a page.